packages/bitcore-node/src/modules/ethereum/api/internalTxTransform.ts

```diff
@@ -0,0 +1,19 @@
+import { Transform } from 'stream';
+import Web3 from 'web3';
+import { MongoBound } from '../../../models/base';
+import { IEthTransaction } from '../types';
+
+export class InternalTxRelatedFilterTransform extends Transform {
+  constructor(private web3: Web3) {
+    super({ objectMode: true });
+  }
+
+  async _transform(tx: MongoBound<IEthTransaction>, _, done) {
+    if (tx.internal && tx.internal.length > 0) {
```

Comment:
I think internal transactions should probably push additional transactions to the stream, not modify the current transaction. 

Also, our ETH node doesn't return internal data, as far as I know